apps/betterangels/src/app/(private-screens)/form/Date.tsx

```diff
@@ -20,6 +20,8 @@ export default function DateComponent(props: IDateProps) {
   const dateValue = watch('date');
   const isDate = expanded === 'Date';
 
+  console.log(dateValue);
```

Comment:
do we want to commit this?

Reply:
we are not using this screen. left to pass lint. didn't want to remove all logic for not used variable